server/datamodel.go

```diff
@@ -307,17 +307,19 @@ type ClientComMessage struct {
 	Del   *MsgClientDel   `json:"del"`
 	Note  *MsgClientNote  `json:"note"`
 
+	// Internal fields, routed only within the cluster.
+
 	// Message ID denormalized
-	id string
+	Id string `json:"-"`
```

Comment:
Do we need `json:"-"` here?

Reply:
Yes. Otherwise it will produce json like 
`{"ctrl": {...}, "id": "123"}`